packages/editor/src/store/actions.js

```diff
@@ -89,15 +82,15 @@ export function __experimentalTearDownEditor() {
  * Returns an action object used in signalling that the latest version of the
  * post has been received, either by initialization or save.
  *
- * @param {Object} post Post object.
- *
- * @return {Object} Action object.
+ * @deprecated Since WordPress 6.0.
  */
-export function resetPost( post ) {
-	return {
-		type: 'RESET_POST',
-		post,
-	};
+export function resetPost() {
```

Comment:
This feels like an action that could potentially be used by third party plugins, it would be good to try to find usage in wpdirectory.net before making this breaking change.

Alternatively, we can also consider deprecating it but keeping its behavior.

Reply:
I searched in wpdirectory.net and haven't found any usage of `resetPost`.

Even if it was used, what would it be supposed to do? The action used to reset the entire post with all attributes (content, title, ...) to a new value, but today it can change just the ID and type. Everything else has moved to the core store. And can `resetPost` be really used to completely reinitialize the editor with an entirely new post ID? I doubt that it's so easy and that setting "current post ID" in editor state is enough.

I can modify the patch to keep the `RESET_POST` reducer behavior -- my main goal is to remove the redundant `resetPost` call from `setupEditor`.

Reply:
Thanks for looking, let's keep it that way and adapt if we ever receive any feedback about this.